lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1970,5 +1970,41 @@ sub status_info {
     return $info;
 }
 
+# a condensed mapping when less states need to be differentiated, e.g. for
+# favicons based on state
+sub simplified_state {
+    my ($self) = @_;
+    my %simplified_state = (
```

Comment:
It would make more sense to have this as a module-wide variable so the hash isn't initialized every time the function is called. (Add `my %SIMPLIFIED_STATE = ( ...` outside of the function.) @kraih Or does Perl "magically" optimize cases like this where the hash isn't modified?

Reply:
Could we use `state` here instead of `my` to avoid the re-initialization?

Reply:
No, Perl does not optimize this case. Yes, you could use `state` too, but personally i prefer a `my %SIMPLIFIED_STATE` at the top of the file.

Reply:
well, we could also use https://metacpan.org/pod/Readonly if you like. But maybe later

Reply:
top of the file, got it, done.